wrappers/core18.go

```diff
@@ -231,71 +255,75 @@ func AddSnapdSnapServices(s *snap.Info, inter interacter) error {
 		// systemd version, where older versions (eg 229 in 16.04) would
 		// error out unless --force is passed, while new ones remove the
 		// symlink and create a new one.
-		enabled, err := sysd.IsEnabled(unit)
-		if err != nil {
-			return err
-		}
-		if enabled {
-			continue
+		if !opts.Preseeding {
+			enabled, err := sysd.IsEnabled(unit)
```

Comment:
So this may be interesting to see in the emulation mode, but doing some little testing, `systemctl --root=foo is-enabled` works directly with filesystem.

Reply:
It doesn't make much sense to check IsEnabled when preseeding because we know we have clean slate. I'd keep this one as is.